ui/pages/confirmations/components/signature-request-original/signature-request-original.component.js

```diff
@@ -1,426 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
I think we'll still need SignatureRequestOriginal to support `eth_signTypedData_v1` aka `eth_signTypedData`. We mentioned it's [deprecated](https://docs.metamask.io/wallet/concepts/signing-methods/#deprecated-signing-methods), but we're currently still supporting it

we can deprecate this one when we remove the "Improve signature requests" setting